automation/taskcluster/lib/tasks.py

```diff
@@ -667,7 +667,7 @@ def _craft_raptor_task(
             "--download-symbols=ondemand",
         ]]
         # Bug 1558456 - Stop tracking youtube-playback-test on motoG5 for >1080p cases
-        if variant.abi == 'arm':
+        if variant.abi == 'arm' and test_name == 'raptor-youtube-playback':
```

Comment:
I'd be happier if we could've specified test-specific options as part of the arguments for `craft_raptor_task(...)`, rather than having a generic function that has branching behaviour based on specific tests.

However, this block existed before, and this is some refactoring that the releng team can investigate further.